config/drupal.settings

```diff
@@ -28,13 +28,17 @@ $databases['migrate']['default'] = $databases['drupal7db']['default'];
 // Custom configuration sync directory under web root.
 $settings['config_sync_directory'] = getenv('CONFIG_SYNC_DIRECTORY');
 
+// Redis Cache.
+// Due to issues with enabling Redis during install/config import. We cannot enable the cache backend by default.
+// Once you have a site/db installed. Enable the Redis module and change the $redis_enabled to true.
+$redis_enabled = false;
```

Comment:
`s/false/FALSE`

I don't think PHPCS runs here, but you know... consistency/habits that die hard etc 😂